pyproject.toml

```diff
@@ -51,6 +51,8 @@ dependencies = [
 
     "pyjwt>=2.6.0",
 
+    "distro",
```

Comment:
Document this dependency a bit more. Mentioning it in the changelog with a link to more info is enough.